src/proxy/command.rs

```diff
@@ -207,6 +209,8 @@ impl DataCmdType {
             b"UNLINK" => DataCmdType::UNLINK,
             b"ZPOPMAX" => DataCmdType::ZPOPMAX,
             b"ZPOPMIN" => DataCmdType::ZPOPMIN,
+            b"BZPOPMAX" => DataCmdType::BZPOPMAX,
+            b"BZPOPMIN" => DataCmdType::BZPOPMIN,
             b"ZREM" => DataCmdType::ZREM,
```

Comment:
Also need to add these two commands here: https://github.com/doyoubi/undermoon/blob/master/src/proxy/command.rs#L437